### PR TITLE
Piper: show how to use Piper TTS with media player in automation

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Piper: show how to use Piper TTS with media player in automation
- Add procedure on how Piper can be used for local text-to-speech processing with a media player
- Addresses the issue mentioned in https://github.com/home-assistant/addons/issues/3034

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/voice_control/using_tts_in_automation.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    title: Your coffee is ready! Enjoy.
+    thumbnail: https://brands.home-assistant.io/_/tts/logo.png
+    media_class: app
+    children_media_class: null
+    navigateIds:
+        - {}
+        - media_content_type: app
+        media_content_id: media-source://tts
+        - media_content_type: provider
+        media_content_id: media-source://tts/tts.piper?message=Your+coffee+is+ready%21+Enjoy.
```
Comment (frenck) [inline source/voice_control/using_tts_in_automation.markdown]: Indentation got lost here, rendering this YAML invalid.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/voice_control/using_tts_in_automation.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    title: Your coffee is ready! Enjoy.
+    thumbnail: https://brands.home-assistant.io/_/tts/logo.png
+    media_class: app
+    children_media_class: null
+    navigateIds:
+        - {}
+        - media_content_type: app
+          media_content_id: media-source://tts
+        - media_content_type: provider
+          media_content_id: media-source://tts/tts.piper?message=Your+coffee+is+ready%21+Enjoy.
```
Comment (frenck) [inline source/voice_control/using_tts_in_automation.markdown]:
```suggestion
      media_content_id: media-source://tts/tts.piper?message=Your+coffee+is+ready%21+Enjoy.
      media_content_type: provider
    metadata:
      title: Your coffee is ready! Enjoy.
      thumbnail: https://brands.home-assistant.io/_/tts/logo.png
      media_class: app
      children_media_class: null
      navigateIds:
          - {}
          - media_content_type: app
            media_content_id: media-source://tts
          - media_content_type: provider
            media_content_id: media-source://tts/tts.piper?message=Your+coffee+is+ready%21+Enjoy.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/voice_control/using_tts_in_automation.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    ![Enter text to be spoken](/images/assist/tts_enter_text.png)
+1. Your text-to-speech action is now ready to be used in your script or automation.
+    ![Enter text to be spoken](/images/assist/tts_action.png)
+1. Save your action.
+1. If you want, you can edit this service in YAML.
+
+    ```yaml
+    service: media_player.play_media
+    target:
+    entity_id: media_player.m5stack_atom_echo_a61920
```
Comment (frenck) [inline source/voice_control/using_tts_in_automation.markdown]:
```suggestion
      entity_id: media_player.m5stack_atom_echo_a61920
```

**Comment by c0ffeeca7** [comment]
Thank you for your help, @frenck

**Review by frenck — APPROVED** [review]

**Pull request merged**